sdcm/cluster.py

```diff
@@ -5098,14 +5103,15 @@ def node_setup(self, node, verbose=False, **kwargs):  # pylint: disable=unused-a
             self.log.info("Don't install anything because bare loaders requested")
             return
 
-        node.create_node_certificate(node.ssl_conf_dir / TLSAssets.CLIENT_CERT,
-                                     node.ssl_conf_dir / TLSAssets.CLIENT_KEY)
-        for src in (CA_CERT_FILE, JKS_TRUSTSTORE_FILE):
-            shutil.copy(src, node.ssl_conf_dir)
-        if self.params.get('client_encrypt'):
+        if self.params.get('client_encrypt') and not self.params.get("cluster_backend").startswith("k8s"):
```

Comment:
Same here. The `k8s` check was needed only in the `tester.py`.
Redundant, not harmful.

Reply:
Got it. Reverted for nodes